consensus/scheme/rolldpos/fsm.go

```diff
@@ -368,14 +368,19 @@ func (m *cFSM) handleStartRoundEvt(_ fsm.Event) (fsm.State, error) {
 	m.ctx.round.proposer = proposer
 	m.ctx.round.timestamp = m.ctx.clock.Now()
 
+	m.produce(m.newCEvt(eInitBlockPropose), 0)
 	// Setup timeout for waiting for proposed block
-	m.produce(m.newCEvt(eInitBlock), 0)
-	m.produce(m.newTimeoutEvt(eProposeBlockTimeout), m.ctx.cfg.AcceptProposeTTL)
+	ttl := m.ctx.cfg.AcceptProposeTTL
```

Comment:
Can we add one validation at config.go to make sure ttl1 + ttl2 + ttl3 <= block interval